exporter/sapmexporter/config.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sapmexporter
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+)
+
+// Config defines configuration for SAPM exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+	Endpoint                      string                   `mapstructure:"endpoint"`
+	AuthToken                     string                   `mapstructure:"auth_token"`
```

Comment:
nit: per comment in the signalfx exporter PR, use `access_token` instead of `auth_token`, see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38#discussion_r348496782

Reply:
nit: add comments to each field here.